src/App.jsx

```diff
@@ -14,70 +15,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState(`Jam`);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const clearGood = () => {
+    setSelectedGood('');
+  };
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  const chooseGood = (good) => {
+    setSelectedGood(good);
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  return (
+    <main className="section container">
+      {selectedGood === '' ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {`${selectedGood} is selected`}
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={clearGood}
+          />
+        </h1>
+      )}
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(item => (
+            <tr
+              key={item}
+              data-cy="Good"
+              className={
+                selectedGood === item ? 'has-background-success-light' : ''
+              }
             >
-              +
-            </button>
-          </td>
+              {selectedGood === item ? (
+                <td>
```

Comment:
Move `<td>` out of the ternary op, so you don't repeat it
```suggestion
              <td>
                {selectedGood === item ? (
```

Reply:
Not fixed, you can only "switch: buttons depending on the condition, it makes no sense to wrap a cell in a condition